src/shared/components/home/emojis-form.tsx

```diff
@@ -267,7 +267,11 @@ export class EmojiForm extends Component<EmojiFormProps, EmojiFormState> {
             {I18NextService.i18n.t("add_custom_emoji")}
           </button>
 
-          <Paginator page={this.state.page} onChange={this.handlePageChange} />
+          <Paginator
+            page={this.state.page}
+            onChange={this.handlePageChange}
+            nextDisabled={false}
```

Comment:
Is there never a time that this needs to be disabled?

Reply:
It needs to be disabled when there are multiple lists (because the API doesn't combine them)